types/underscore/underscore-tests.ts

```diff
@@ -486,6 +486,8 @@ _(numberArray).map((value, key, collection) => {
     return value;
 });
 
+_.get({ a: 1 }, ['a'], 2);
```

Comment:
Would be nice to also add calls without a default and with a string path instead of an array path. Also, please add an `$ExpectType` like in the other tests.